src/types/fetchers.ts

```diff
@@ -16,6 +17,10 @@ export type StringMultiProcessorResult = string | undefined;
 
 export type OnChainDataFetcherResult = string | number;
 
+export type NumberOrUndefined = number | undefined;
+
+export type StringOrUndefined = string | undefined;
+
 // TODO: refactor this type
 export type FeedFetcherInterfaceResult =
```

Comment:
let's also cleanup this, probably we can reuse this new types `NumberOrUndefined`...

Reply:
We decided to do it in another PR